src/components/Pagination/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import ButtonIcon from '../ButtonIcon';
+import LeftArrow from './leftArrow';
+import RightArrow from './rightArrow';
+import PageButtons from './pageButtons';
+import './styles.css';
+
+export default function Pagination(props) {
+    const { pages, activePage, onChange, className, style } = props;
+
+    const getContainerClassNames = () => classnames('rainbow-pagination_container', className);
+
+    const isFirstItemSelected = activePage === 1;
+    const isLastItemSelected = activePage === pages;
+
+    return (
+        <div className={getContainerClassNames()} style={style}>
+            <ButtonIcon
+                variant="border-filled"
+                className="rainbow-pagination_button rainbow-pagination_button-icon"
+                icon={<LeftArrow />}
+                onClick={() => onChange(activePage - 1)}
```

Comment:
here pass the event too and the same above

Reply:
I will say that the we should use a hidden input inside then we change the value on the input in order to trigger the input change event